### PR TITLE
Header up2

### PR DESCRIPTION
**Pull request by tsmandax**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,7 +16,27 @@
     <![endif]-->
   </head>
   <body>
-    <header></header>
+    <header>
+      <div class="navbar navbar-static-top header-up">
```
Comment (khos-erdene) [inline index.html]: enen deer navbartai classiig bitgii ashigla. Ene classiig yrunhiiduu menun deer l ashigladag. Eniig taviad bgaa bolhoor min-height zaaval tavih asuudal garj ireed bn

File: index.html [hunk 2]
```diff
@@ -16,7 +16,27 @@
...
+        <div class="container-fluid">
+          <div class="pull-left">
```
Comment (khos-erdene) [inline index.html]: ugaasaa l web lefteesee eheldeg bolhoor eniig bicheh shaardlaga bhgui. Iluu uildel

File: index.html (hunk 1 above)
Comment (tsmandax) [reply index.html]: tegehleer enenii orond custom class bichej ogno gesen uguu?

File: index.html [hunk 3]
```diff
@@ -16,7 +16,27 @@
...
+            <ul class="nav navbar-nav header-up-nav">
```
Comment (khos-erdene) [inline index.html]: ene dahiad l nav-tai classiig ashiglachaad bn ene bolkuee. ene ni deer ug ni list-inline gsen class ashiglah heregtei gj umnuh review deer bichsenshd

File: index.html [hunk 4]
```diff
@@ -16,7 +16,27 @@
...
+              <li class="hidden-xs hidden-sm"><a><i class="fa fa-flag" aria-hidden="true"></i> Монгол улс, Улаанбаатар хот Баянзүрх дүүрэг,22-р хороо</a></li>
```
Comment (khos-erdene) [inline index.html]: a href="" deer ym bichehiig sudlaarai gsen ene 3 deer bicheegui bn. a href="tel:" mailto geh met bgaa

File: index.html [hunk 5]
```diff
@@ -16,7 +16,27 @@
...
+              <li class="hidden-xs"><a><i class="fa fa-phone" aria-hidden="true"></i> +976 99195729</a></li>
+              <li><a><i class="fa fa-envelope-o" aria-hidden="true"></i> helloworld@gmail.com</a></li>
+            </ul>
+          </div>
+          <div class="pull-right">
+            <ul class="nav navbar-nav header-up-social">
```
Comment (khos-erdene) [inline index.html]: bas l buruu class name

File: index.html [hunk 6]
```diff
@@ -16,7 +16,27 @@
...
+              <li><a href="https://www.facebook.com/SysAndCoTech/"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
```
Comment (khos-erdene) [inline index.html]: ene a taguud deer target blank ashiglah heregtei bdag. Tgehgui bol shuud huudas ni soligdochdog

File: css/custom.css [hunk 7]
```diff
@@ -1,6 +1,38 @@
...
+}
+.header-up-nav {
+  margin: 0;
+}
+.header-up-nav>li>a {
+  color: #5f5f5f;
+  padding-top: 5px!important;
+  padding-bottom: 5px!important;
+}
+.header-up-nav>li>a>i {
```
Comment (khos-erdene) [inline css/custom.css]: iconii araas joohon zai avsan ni deer hetehii naaldaad bn

File: css/custom.css [hunk 8]
```diff
@@ -1,6 +1,38 @@
...
+  border: 0;
+  font-size: 13px;
+  background-color: #f6f9fe;
+}
+.header-up-nav {
+  margin: 0;
+}
+.header-up-nav>li>a {
+  color: #5f5f5f;
+  padding-top: 5px!important;
```
Comment (khos-erdene) [inline css/custom.css]: nav-tai classiig ashiglahgui hiih ystoi uchraas. Ene deer important bichehgui. Bas !importantiin urd 1 space avdag.

File: css/custom.css [hunk 9]
```diff
@@ -1,6 +1,38 @@
+.header-up {
+  margin-bottom: 0;
+  min-height: 0;
```
Comment (khos-erdene) [inline css/custom.css]: yr ni tsaashid min-height reset hiih gj bgaa bol min-height: inherit bh ystoi

File: css/custom.css [hunk 10]
```diff
@@ -1,6 +1,38 @@
...
+  border: 0;
+  font-size: 13px;
+  background-color: #f6f9fe;
+}
+.header-up-nav {
```
Comment (khos-erdene) [inline css/custom.css]: enen deer mouse ochihoor background-toi bgaa ni mash sonin haragdaj bn. Ard taliin socialuud bol goy bn

File: css/custom.css [hunk 11]
```diff
@@ -1,6 +1,38 @@
...
+}
+.header-up-nav>li>a {
+  color: #5f5f5f;
+  padding-top: 5px!important;
+  padding-bottom: 5px!important;
+}
+.header-up-nav>li>a>i {
+  color: blue;
+}
+.header-up-social>li>a {
```
Comment (khos-erdene) [inline css/custom.css]: arai l hoorondoo hol zaitai bgaad bn padding ihtei

File: css/custom.css [hunk 12]
```diff
@@ -1,6 +1,38 @@
...
+  color: #5f5f5f;
+  padding-top: 5px!important;
+  padding-bottom: 5px!important;
+}
+.header-up-social>li>a:hover {
+  color: blue;
+}
 /* Tablet bolon jijig delgetstei computeruud */
 @media (min-width: 768px) and (max-width:960px) { 
+  .nav>li{
```
Comment (khos-erdene) [inline css/custom.css]: ene mash buruu uildel

File: css/custom.css [hunk 13]
```diff
@@ -1,6 +1,38 @@
...
+    float: left;
+  }
 }
 /* Gar utasnuud */
 @media (max-width: 767px) {
+  .nav>li{
```
Comment (khos-erdene) [inline css/custom.css]: deed taliin commenttoi adilhan

File: index.html [hunk 14]
```diff
@@ -16,7 +16,33 @@
     <![endif]-->
   </head>
   <body>
-    <header></header>
+    <header>
+      <div class="header-up">
+        <div class="container-fluid">
+          <div class="row">
+            <div class="col-xs-7">
```
Comment (khos-erdene) [inline index.html]: ene gridiig butneer ni bichvel zugeer bdag xl deer ted md deer ted sm deer ted xs ted gd bicheerei

**Pull request merged**